odo/__init__.py

```diff
@@ -1,73 +1,37 @@
-from __future__ import absolute_import, division, print_function
+from __future__ import absolute_import as _
```

Comment:
I don't think this is needed. Many projects export these names without confusing people, for example: numpy does this.